entry.js

```diff
@@ -25,9 +28,8 @@ const deps = {
 }
 
 export { deps }
-
-// version string.
-export { version } from './package.json'
+const version = pkgInfo.version
+export { version }
```

Comment:
Question here. Is this piece necessary? It seems to be adding an extra variable that is otherwise unused.

Reply:
I know this looks weird, but it's there to remove a webpack warning.

Reply:
Lol.  I hear you. Doing odd things to suppress warnings is one of the favorite pastimes.